tests/test_processors.py

```diff
@@ -456,6 +456,17 @@ def test_obtains_exc_info_on_True(self):
         else:
             pytest.fail("Exception not raised.")
 
+    def test_obtains_exc_info_on_number(self):
```

Comment:
your cases are fine but please use pytest’s parametrization for it – sorry I should have been more clear: http://docs.pytest.org/en/latest/parametrize.html (pro tip: they don't have to be tuples if it’s just one arg).